service/frontend/adminHandler.go

```diff
@@ -1391,10 +1660,10 @@ func (adh *adminHandlerImpl) ListDynamicConfig(ctx context.Context, request *typ
 	}, nil
 }
 
-func checkValidKey(keyName string) (dynamicconfig.Key, error) {
-	keyVal, ok := dynamicconfig.KeyNames[keyName]
-	if !ok || keyVal == dynamicconfig.UnknownKey {
-		return dynamicconfig.UnknownKey, errors.New("invalid dynamic config parameter name")
+func checkValidKey(keyName string) (dc.Key, error) {
+	keyVal, ok := dc.KeyNames[keyName]
+	if !ok || keyVal == dc.UnknownKey {
+		return dc.UnknownKey, errors.New("invalid dynamic config parameter name")
```

Comment:
nit: not directly related to this change, but it is always good to know, what exactly was passed, we can improve error message like this:
```suggestion
		return dc.UnknownKey, fmt.Errorf("invalid dynamic config parameter name: %q")
```